### PR TITLE
Ecf 119231421 multiple consents

### PR DESCRIPTION
**Pull request by ericcf**
Most of the interesting code is in the first commit. The rest is fixing broken specs, etc.

### CODE REVIEW

File: spec/features/consent_form_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  let(:sign_in_page) { Pages::SignIn.new }
+  let(:consent_page) { Pages::Consent.new }
+
+  scenario "a Participant consents", js: true do
+    sign_in_page.visit
+    sign_in_page.sign_in
+
+    expect(consent_page).to have_content "PALO ALTO UNIVERSITY CONSENT"
+
+    consent_page.consent_yes
```
Comment (michaelwehrley) [inline spec/features/consent_form_spec.rb]: Is this a smoke test?

Comment (ericcf) [inline spec/features/consent_form_spec.rb]: Ha, good catch. Got everything working and forgot to write the last part of this spec.

File: spec/jobs/follow_up_notification_spec.rb [hunk 2]
```diff
@@ -11,6 +11,7 @@
   end
 
   def consent(participant, is_consented)
+    ConsentResponse.where(participant: participant).destroy_all
```
Comment (michaelwehrley) [inline spec/jobs/follow_up_notification_spec.rb]: Quick question, it seems that participant `has_one` consent response.  It seems here that a participant could have many consent responses.  Could you please enlighten me?  Thanks.

File: spec/pages/consent.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+module Pages
+  class Consent
+    include Capybara::DSL
```
Comment (michaelwehrley) [inline spec/pages/consent.rb]: 👍

File: spec/jobs/follow_up_notification_spec.rb (hunk 2 above)
Comment (ericcf) [inline spec/jobs/follow_up_notification_spec.rb]: `has_one` doesn't prevent you from creating multiple associated records. This is just a quick and dirty way to ensure any/all associations are removed before generating more.

File: vendor/assets/javascripts/bind-polyfill.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+if (!Function.prototype.bind) {
```
Comment (michaelwehrley) [inline vendor/assets/javascripts/bind-polyfill.js]: Good comments.

**Comment by michaelwehrley** [comment]
:shipit: when ready!  good work.

File: spec/jobs/follow_up_notification_spec.rb (hunk 2 above)
Comment (michaelwehrley) [inline spec/jobs/follow_up_notification_spec.rb]: true, but you also have a unique participant_id constraint: `validates :participant_id, uniqueness: true`

**Pull request merged**